validator/validate_api.py

```diff
@@ -293,22 +299,27 @@ def validation_api_entry():
 
     response = args.get("response", None)
     uid = args.get("uid", None)
+    feature_weights_set_id= args.get("feature_weights_set_id", 'd3732be6-a759-43aa-9e1a-3e9bd94f8b6b')
```

Comment:
let's get the default key from the app settings

Reply:
Oh, and validate that the key exists in current_app.df['feature_weights'], otherwise throw some sort of error.